joplin/base/migrations/0006_auto_20200205_2143.py

```diff
@@ -0,0 +1,40 @@
+# Generated by Django 2.2.9 on 2020-02-05 21:43
```

Comment:
It'd be nice to get this down to 1 migration, it's not clear what's different in the 3